packages/varnish/src/theme/theme.ts

```diff
@@ -235,8 +248,8 @@ export const getVarnishDesignTokens = (): ThemeOptions => {
         active: color2.B3.hex,
         hover: color2.B3.hex,
         selected: color2.B3.hex,
-        disabled: color2.B3.hex,
-        disabledBackground: color2.N5.hex,
+        disabled: color2.B6.hex,
```

Comment:
lets speak with @samstuesser 
but i am thinking we should consider adding specific colors to extended palette (like background.dark) instead of b6 and n6...
unless b6 and n6 are used commonly.

further... i am wondering what we might be doing wrong with the colors, because the disabled button should be transforming the color of the button, not getting a specific override..

what does this look like for button color=warning? or color=error?

Reply:
ok, i spoke with @samstuesser 
1- dont add B6 nor N6
2- dont ever add any colors to the colors list :-P
3- the color you are seeing is B3 with a transparency set by mui
4- WHY its not getting set when we run the code is odd...
4b- but we should not be nor need to be setting it
5- lets talk in person

Reply:
we decided to revisit this in a separate set of changes so resolving this comment on this pr